discovery_engine_core/web-api/src/models.rs

```diff
@@ -59,18 +59,11 @@ impl AiDocument for Document {
 }
 
 /// Represents an article that is stored and loaded from local json file.
-#[derive(Debug, Clone, Serialize, Deserialize)]
-pub(crate) struct Article {
-    pub(crate) id: Uuid,
-    pub(crate) description: String,
-}
+pub(crate) type Article = HashMap<String, serde_json::Value>;
```

Comment:
:art: if space/performance is an issue, then we could also use an enum instead of strings as keys for the hashmap. another nice side effect is that typos in the keys would be caught at compile time instead of at runtime.